### PR TITLE
Added --ANALYZE option

### PR DESCRIPTION
**Pull request by na6vj**
You can now analyze the generated data using the --analyze option.

### CODE REVIEW

File: querymonitor/README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## Synopsis
 Runs (as a daemon) and periodically collects live query info into csv (.gz) files.
-Personably Identifyable information (PII) is removed from each query by truncating the WHERE clause.
+Personably Identifyable information (PII) can be removed from each query by truncating the WHERE clause.
+Built-in --ANALYSIS mode.
```
Comment (eugeneckim) [inline querymonitor/README.md:6]: The option says "--ANALYSIS" here but line 62 says "--ANALYZE"

File: querymonitor/querymonitor.pl [hunk 2]
```diff
@@ -321,5 +332,108 @@ sub unixtime_to_printable{
...
+.version
+.header off
+CREATE TABLE q (ts integer,clientHost,clientPort,elapsedMillis integer,id,keyspace,nodeName,privateIp,query,type);
+CREATE VIEW summary as select datetime((ts/600)*600,'unixepoch','-4 Hours') as EDT,
+    sum(case when query LIKE '% system.%'  then 1 else 0 end) as systemq,
+        sum(case when query LIKE '% system.%' then 0 else 1 end) as cqlcount,
+        sum(case when query LIKE '% system.%' and elapsedmillis > 120 then 1 else 0 end) as sys_gt120,
+        sum(case when query NOT LIKE '% system.%' and elapsedmillis > 120 then 1 else 0 end) as cql_gt120,
+        sum(case when query LIKE '% system.%' and nodeName
+            in ('yb-prod-CAS-PROD-Primary-n2','yb-prod-CAS-PROD-Primary-n3','yb-prod-CAS-PROD-Primary-n1','yb-prod-CAS-PROD-Primary-n5',
```
Comment (eugeneckim) [inline querymonitor/querymonitor.pl:396]: As a sanity check, is this intentional that we're hard-coding in node names from customer?

File: querymonitor/README.md (hunk 1 above)
Comment (na6vj) [reply querymonitor/README.md:6]: Fixed in next check-in

File: querymonitor/querymonitor.pl (hunk 2 above)
Comment (na6vj) [reply querymonitor/querymonitor.pl:396]: Reduced dependency on hard coded names in the next checkin.

The plan is  to capture Live Universe config (node/region/zone) into the file to be analyzed such that it does not need to be hard-coded.

Comment (ionthegeek) [reply querymonitor/querymonitor.pl]: When this gets merged, it should probably be squashed. We don't want customer info in the git history if we can help it.

See: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges

**Comment by na6vj** [comment]
Resubmitting PULL request - Code has been restructured removing all customer references.
Please approve.

**Review by eugeneckim — APPROVED** [review]
El Gee Two Mi

**Pull request merged**